uspace/app/bdsh/cmds/modules/unalias/unalias.c

```diff
@@ -88,15 +89,17 @@ int cmd_unalias(char **argv)
 
 			return CMD_SUCCESS;
 		}
+	}
 
+	size_t i;
+	for (i = 1; argv[i] != NULL; i++) {
 		alias_link = odict_find_eq(&alias_dict, (void *)argv[i], NULL);
 
 		if (alias_link == NULL) {
 			cli_error(CL_ENOENT, "%s: No alias '%s' found\n", cmdname, argv[i]);
-			return CMD_FAILURE;
+		} else {
+			free_alias(alias_link);
 		}
-
-		free_alias(alias_link);
 	}
 
 	return CMD_SUCCESS;
```

Comment:
Didn't you want to return `CMD_FAILURE` when an error was encountered?
I don't care either way, just checking if it's intended.

Reply:
I completely forgot about that. Thx!!